client/app/stores/router_store.coffee

```diff
@@ -437,6 +448,10 @@ class RouterStore extends Store
         _URI
 
 
+    getMessagesPerPage: ->
+        MSGBYPAGE
```

Comment:
It will be usefull for `router_store.specs` but you're right, it is no used, but should be... I remove it?